lib/proxymode.pm

```diff
@@ -127,4 +127,73 @@ sub redirect_serial {
     select_console "root-console";
     $self->reset_curr_serialdev();
 }
+
+sub get_consoledev {
+    if (get_var("XEN") || check_var("HOST_HYPERVISOR", "xen")) {
+        script_run("clear");
+        my $prd_version = script_output("cat /etc/SuSE-release");
```

Comment:
just FYI, the new way is `/etc/os-release`. Don't rely on `/etc/SuSE-release` in the future